src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+class Square extends Figure {
+    private double side;
+
+    public Square(Color color, double side) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create a variable for 'side' and pass it to the constructor for better code readability.